api/v1/openapi.yaml

```diff
@@ -2445,10 +2445,16 @@ definitions:
         additionalProperties:
           type: object
       GSOMaxSize:
-        description: Maximum GSO size on workload facing devices
+        description: Maximum IPv6 GSO size on workload facing devices
```

Comment:
I think I would have made GSOMaxSize the ipv4 defult but hey doesn't matter in the end.

Reply:
ah but, I guess that would break existing stuff so ok

Reply:
Agree, I extended this here as users might have kernels with IPv6 BIG TCP but not yet IPv4 BIG TCP. The other thought  that would be good in future is to have v4/v6 specific tunables for Cilium.